greek/put_socks_off/put-socks-off.md

```diff
@@ -2,5 +2,8 @@
 άραγε παρόμοια αισθητήρια όπως και εσύ; Βλέπει, ακούει, μυρίζει; 
 Ας δούμε λοιπόν πως αντιδράει στην μοχθηρή μυρωδιά μιας
 χιλιο-φορεμένης καλτσας...
-... ίσως το ζαχαρωτό να αρχίσει να δακρύζει. Τι θα μας έδειχνε αυτό;
+... ίσως το ζαχαρωτό να αρχίσει να δακρύζει. Ενδεχομένως να άρχιζε το φτάρνισμα. Ή ακόμα χειρότερα το βήχα... Τι θα μας έδειχνε αυτό;
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.